databricks_sql_connector_core/src/databricks/sql/client.py

```diff
@@ -118,7 +125,7 @@ def __init__(
 
                 from databricks.sql.experimental.oauth_persistence import OAuthPersistence, OAuthToken
                 class MyCustomImplementation(OAuthPersistence):
-                    def __init__(self, file_path):
+                    def __init__.py(self, file_path):
```

Comment:
Same here - `__init__` is a method name, not a file name. Also, how did this code even run? It has to throw a syntax error